src/libs/actions/Report.js

```diff
@@ -294,7 +295,15 @@ function addActions(reportID, text = '', file) {
         },
     ];
 
+    const failureReport = {
+        lastMessageText: prevVisibleMessageText,
+    };
     const failureData = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: failureReport,
+        },
```

Comment:
This caused a [regression](https://github.com/Expensify/App/issues/26047). If we send a message after the attachment and delete that message the LHN would appear again as `[Attachment]`. Not exactly a bug from this PR but it caused the inconsistency as previously seeing `[Attachment]` in LHN was the expected results.